index.bs

```diff
@@ -1784,6 +1788,57 @@ endpoint is verified to be a WebTransport endpoint; thus, the Web application
 cannot distinguish between a non-existing endpoint and the endpoint that is not
 willing to accept connections from the Web.
 
+## Authentication using Certificate Hashes {#certificate-hashes}
+
+In the vast majority of cases, a user agent authenticates a TLS connection
+between itself and a remote endpoint by verifying the validity of the TLS
+server certificate provided against the server name in the URL; this approach
```

Comment:
I think that you are missing a key component in this definition: the certification path construction to trust anchors defined by the CA/Browser Forum.  It's not that the certificate is valid (that's almost trivial) and contains the right name (actually trivial), but that the certificate chains to a trust anchor.

Reply:
From my perspective, the role of WebPKI as far as immediate network logic is concerned is establishing a validity of a `(server identity, public key)` pair; to some extent, the fact that we have trust anchors at all is an implementation detail, as the key observation here is that fingerprint-based verifier sidesteps this problem by making the first and the second elements of this pair essentially the same thing.